cmd/minishift/cmd/start.go

```diff
@@ -325,6 +325,8 @@ func startHost(libMachineClient *libmachine.Client) *host.Host {
 		HostOnlyCIDR:     viper.GetString(configCmd.HostOnlyCIDR.Name),
 		ShellProxyEnv:    shellProxyEnv,
 	}
+	minishiftConfig.InstanceConfig.VMDriver = machineConfig.VMDriver
```

Comment:
maybe not persist, but actually wrap around the viper read action, as this only lives in config.json.

Reply:
@gbraad Why are we adding the VMDriver information to instance config, because Viper will not read it from there e.g. `viper.GetString(configCmd.VmDriver.Name)`

Reply:
NVM, I see  pkg/minishift/config/functions.go where you have used it.

Reply:
Config implementation is part of #1796